### PR TITLE
Rule: Follina and DogWalk exploit msdt.exe loading sdiageng.dll

### PR DESCRIPTION
**Pull request by greg-workspace**
Hello, 
I close the old pull request: process creation of  "sdiagnhost.exe". Detecting "sdiageng.dll" is more precise. Besides, this rule is able to detect both Follina and DogWalk attack.

### CODE REVIEW

**Comment by Neo23x0** [comment]
In the `ImageLoaded` we get a full path, right? 
Can we prepend a `\`?

File: rules/windows/image_load/image_load_msdt_sdiageng.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - https://www.securonix.com/blog/detecting-microsoft-msdt-dogwalk/
+date: 2022/06/17
+logsource:
+  category: image_load
+  product: windows
+detection:
+  selection_img:
+    Image|endswith: '\msdt.exe'
+  selection_load:
+    ImageLoaded|endswith: 'sdiageng.dll'
```
Comment (Neo23x0) [inline rules/windows/image_load/image_load_msdt_sdiageng.yml:16]: ImageLoaded|endswith: '\sdiageng.dll'

**Review by greg-workspace — COMMENTED** [review]
Hello, can any one share us what kind of tests to verify the rule? E.g., test-sigma, sigmalint, Zircolite, etc. In addition, do you recommend any Github project  to help us verify the rules?
Thank you.

File: rules/windows/image_load/image_load_msdt_sdiageng.yml (hunk 1 above)
Comment (greg-workspace) [reply rules/windows/image_load/image_load_msdt_sdiageng.yml:16]: Yes, it is better.

**Comment by Neo23x0** [comment]
Doesn't a `make test` in the root of the repository work for you?

**Pull request merged**